opts/opts.go

```diff
@@ -277,7 +283,7 @@ func ValidateExtraHost(val string) (string, error) {
 }
 
 func ValidateLabel(val string) (string, error) {
-	if strings.Count(val, "=") != 1 {
+	if strings.Count(val, "=") < 1 {
 		return "", fmt.Errorf("bad attribute format: %s", val)
 	}
```

Comment:
I see no reason to restrict the label name.  Although Spaces always cause problems.   :^(